index.js

```diff
@@ -4,15 +4,42 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+const tName = document.querySelector("#name")
+const nickname = document.querySelector("#nickname")
+const photo = document.querySelector("#photo")
+tName.innerHTML = traveler.name;
+nickname.innerHTML = traveler.nickname;
+photo.src = traveler.photo;
+photo.alt = traveler.name
+console.log('This is what the traveler object looks like: ', traveler)
 
 
 /***** Deliverable 4 *****/
-
+traveler.animalSightings.forEach(function (value, key){
+    var li = document.createElement("li");
+    var p = document.createElement("p");
+    var a = document.createElement("a")
+    var pText = document.createTextNode(value.description)
+    p.append(pText)
+    var img = document.createElement("img");
+    img.src = value.photo
+    img.alt = value.species
+    a.href = value.link
+    a.target = "_blank" 
+    var aText = document.createTextNode("Here is a video about the " + value.species + "species!")
+    a.append(aText)
+    li.setAttribute('data-id' , value.id);
```

Comment:
Good work with this deliverable! For setting dataset attributes, you can also write:
```suggestion
    li.dataset.id = value.id
```